### PR TITLE
Add/grunion alert bad emails

### PR DESCRIPTION
**Pull request by samhotchkiss**
take two of #1635

### CODE REVIEW

File: modules/contact-form/grunion-contact-form.php [hunk 1]
```diff
@@ -177,9 +177,9 @@ function grunion_check_notification_emails() {
 	 */
 	function grunion_show_invalid_email_message() {
 		if ( get_option( 'grunion_display_email_error_notice' ) ) { ?>
-			<script>alert( '<?php echo get_option('grunion_display_email_error_notice'); ?>' );</script>
+			<script>alert( '<?php echo htmlspecialchars( get_option('grunion_display_email_error_notice'), ENT_QUOTES ); ?>' );</script>
```
Comment (dereksmart) [inline modules/contact-form/grunion-contact-form.php]: let's use `esc_html()` here instead.
also missing whitespace

**Comment by dereksmart** [comment]
I updated this so it actually works and now uses admin notices instead of js alerts.  The notice will only show one time per post save/update/publish if there are bad email addys found.

File: to-test.txt [hunk 2]
```diff
@@ -32,6 +32,7 @@ To Test: See example filter useage here - https://github.com/Automattic/jetpack/
 To Test: Once you have a language pack installed (chosen in Settings > General), you may get REST data translated like so:
 https://public-api.wordpress.com/rest/v1/sites/$blog/post-formats/?pretty=true&locale=$locale
 
+
```
Comment (zinigor) [inline to-test.txt]: Do we need this whitespace?

Comment (dereksmart) [inline to-test.txt]: nope, not sure how that got there.

**Comment by zinigor** [comment]
Read the code, tested, have a few questions: do y'all think it would be better to issue errors on saving the form rather than on saving the post? This way we will be able to deal with two problems instead of one - the error is too late, and the post/page gets saved with the bad email anyway.

**Comment by dereksmart** [comment]
I think there was a reason I did it this way (back in #1635), but I can't remember why.

Punting to 3.7 so we can look closer at it next release.

**Comment by samhotchkiss** [comment]
@dereksmart -- shall we punt again?

**Comment by zinigor** [comment]
I'm sorry to say that there's no way this will make it into 3.7.

**Comment by dereksmart** [comment]
Closing in favor of the js solution in #2805

**Pull request closed without merging**